.github/workflows/lint.yml

```diff
@@ -103,6 +103,13 @@ jobs:
           crate: cargo-deny
           version: 0.18.3
           locked: true
+      - name: Install cargo-about
+        if: ${{ runner.environment != 'self-hosted' }}
+        uses: baptiste0928/cargo-install@v3
+        with:
+          crate: cargo-about
+          version: 0.8.2
+          locked: true
```

Comment:
cc @delan we will need this in runners.

Reply:
Probably best to remove the `if` for now, and always install it. I added cargo-about to the list in https://github.com/servo/ci-runners/issues/54